FantasmoSDK/Classes/Network/FMRestClient.swift

```diff
@@ -55,17 +31,18 @@ struct FMRestClient {
     ///   - error: Error closure
     static func post(_ endpoint: FMApiRouter.ApiEndpoint,
                      parameters: [String : String],
-                     imageData: Data,
+                     imageData: Data? = nil,
                      token: String?,
                      completion: RestResult? = nil,
                      error: RestError? = nil) {
-        
         let request = Self.requestForEndpoint(endpoint, token: token)
         log.info(String(describing: request.url), parameters: parameters)
         
         var data = Data()
         data.appendParameters(parameters)
-        data.appendImage(imageData)
+        if let theImageData = imageData {
```

Comment:
This is unrelated to the task at hand.

Reply:
Reason is explained in https://github.com/fantasmoio/Fantasmo-iOS-SDK/pull/29#discussion_r670313677